pype/plugins/global/publish/extract_burnin.py

```diff
@@ -236,6 +236,14 @@ def main_process(self, instance):
                         decompressed_dir,
                         input_file)
 
+                # Passing crf data to burnin script.
+                cmd = new_repre.get("ffmpeg_cmd", "")
```

Comment:
Better send ffmpeg cmd arguments to burnin script which should handle that. In that case it would be possible to check for more information. But definitely don't add id to `burnin_data` which is used for filling of data in burnins not for metadata of it's creation.

Reply:
Since h264 and h265 are "distribution codecs" and review with burnin will most likely not be a final delivery, maybe even just having default crf for the burnin encode would be acceptable?

Of course passing crf from the first encode would be awesome.

Reply:
> Since h264 and h265 are "distribution codecs" and review with burnin will most likely not be a final delivery, maybe even just having default crf for the burnin encode would be acceptable?

The point of making the review and burnin the same quality is so you dont get unexpected quality on your Ftrack reviews. Burnin is what is ultimately uploaded and reviewed on Ftrack, but the quality is currently not in sync with the review (which you have control over in the config/settings)

Reply:
> Better send ffmpeg cmd arguments to burnin script which should handle that. In that case it would be possible to check for more information. But definitely don't add id to burnin_data which is used for filling of data in burnins not for metadata of it's creation.

Agree with this. Will amend next week. This is the direction I've taken #2070